lib/explorer/data_frame.ex

```diff
@@ -711,7 +711,7 @@ defmodule Explorer.DataFrame do
 
   defp normalise_entry("s3://" <> _rest = entry, config) do
     config = s3_config(config)
```

Comment:
Now that we have a S3 config from env, should we lazy load it from the environment?

Reply:
I removed this function and delegate all the config to the `parse/2` function.